snippets/formToObject.md

```diff
@@ -0,0 +1,25 @@
+### formToObject
+
+Serializes a form into an object.
+
+First we transform the `form` into `FormData`, then we convert it into an `array` and from the `array` we collect an `object`
+
+```js
+const formToObject = form =>
+  Array.from(new FormData(form))
+    .reduce((acc, [key, value]) => ({
+      ...acc,
+      [key]: value,
+    }), {})
+```
+
+```html
```

Comment:
Including an `html` example here is not necessary and makes the snippet noisy. It's easy to understand the context from the JS result below.

Reply:
ok, i remove it.